addons/kudo/0.9.x/kudo-1.yaml

```
@@ -0,0 +1,27 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: kudo
+  labels:
+    kubeaddons.mesosphere.io/name: kudo
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.9.0"
```

Revisions need to have a `-${revision_number}` for the current version:

```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "0.9.0-1"
```